Imagekit/Helper/ImagekitParams.cs

```diff
@@ -0,0 +1,148 @@
+﻿// <copyright file="ImagekitParams.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace Imagekit
+{
+    public partial class BaseImagekit<T>
+    {
+        public T Path(string value)
+        {
+            return this.Add("path", value);
+        }
+
+        public T Src(string value)
+        {
+            return this.Add("src", value);
+        }
+
+        public T UrlEndpoint(string value)
+        {
+            return this.Add("urlEndpoint", value);
+        }
+
+        public T Url(Transformation value)
+        {
+            return this.Add("transformation", value);
+        }
+
+        public T QueryParameters(params string[] value)
+        {
+            return this.Add("queryParameters", value);
+        }
+
+        public T TransformationPosition(string value)
+        {
+            return this.Add("transformationPosition", value);
+        }
+
+        public T Signed(bool value = true)
+        {
+            return this.Add("signed", value);
+        }
+
+        public T ExpireSeconds(int value)
+        {
+            return this.Add("expireSeconds", value);
+        }
+
+        public T Limit(int value = 1000)
+        {
+            return this.Add("limit", value);
+        }
+
+        public T Skip(int value = 0)
+        {
+            return this.Add("skip", value);
+        }
+
+        public T Name(string value)
+        {
+            return this.Add("name", value);
+        }
+
+        public T IncludeFolder(bool value = false)
+        {
+            return this.Add("includeFolder", value);
+        }
+
+        public T Tags(string value)
+        {
+            return this.Add("tags", value);
+        }
+
+        public T Tags(params string[] value)
+        {
+            return this.Add("tagsList", value);
+        }
+
+        public T FileType(string value = "all")
+        {
+            return this.Add("fileType", value);
+        }
+
+        public T Sort(string value)
+        {
+            return this.Add("sort", value);
+        }
+
+        public T SearchQuery(string value)
+        {
+            return this.Add("searchQuery", value);
+        }
+
+        public T FileName(string value)
+        {
+            return this.Add("fileName", value);
+        }
+
+        public T UseUniqueFileName(bool value = true)
+        {
```

Comment:
`useUniqueFileName` default true seems to be not working. If we don't pass this option in upload request and setting is on in the dashboard, image gets uploaded with exact name without random string in same.

Reply:
The default value for all parameters should match the value mentioned in docs. For this parameter the default value is false.

Reply:
Not working as expected. The default value for this param seems to be `true` as per docs. https://docs.imagekit.io/api-reference/upload-file-api/server-side-file-upload